WebARKit/WebARKitTrackers/WebARKitOpticalTracking/WebARKitTracker.cpp

```diff
@@ -30,7 +30,7 @@ class WebARKitTracker::WebARKitTrackerImpl {
         }
         _camera->setupCamera(frameWidth, frameHeight);
         _camera->printSettings();
-        m_camMatrix = cv::Mat(3, 3, CV_64FC1, _camera->getCameraData().data());
```

Comment:
This seems to not correctly filling the values of the camera matrix.

Reply:
This probably because the `cv::Mat` is a `3 x 3` matrix, instead the underlying type of _camera->getCameraData() is a undidemensional array (std::array<double, 9>)

Reply:
Indeed this was the cause...